python/lsst/ts/athexapod/controller.py

```diff
@@ -54,64 +56,52 @@ class ATHexapodController:
 
     """
 
-    def __init__(self, log=None):
-        self.host = "127.0.0.1"
-        self.port = 50000
-        self.timeout = 2.0
-        self.long_timeout = 30.0
-
-        self.reader = None
-        self.writer = None
+    def __init__(self, log: None | logging.Logger = None) -> None:
+        self.host: str = "127.0.0.1"
+        self.port: int = 50000
+        self.timeout: float = 2.0
+        self.long_timeout: float = 30.0
```

Comment:
Is this set somewhere outside of this class? If yes, make it an attribute of the dunder init method as well. If not, please don't hard code magic numbers and create a constant for it.